utils/decode_kallsyms.py

```diff
@@ -0,0 +1,45 @@
+import bisect
+import re
+
+
+def binary_search_recursive(array, element):
+    return bisect.bisect_right(array, element, key=lambda y: int(y.split()[0], 16)) - 1
+
+
+def cut_addresses_from_log_line_kernel_callstack(line):
+    my_regex = re.compile('0x([0-9a-f]{16})')
+    return my_regex.findall(line)
+
+
+def get_lines_from_log(file_path):
+    file_content = dict()
```

Comment:
dict is the best format for Python to use, and in this case, it doesn't really change much the logic on the purpose of this dictionary usage, since we want to print both the line as it is in the log, and the decoded kernel callstack

Reply:
This was just a nitpick. Dict is okay, if you like it more.